MANUAL.md

```diff
@@ -297,6 +297,16 @@ The `web_server` section lets you tune the performance of your RMT server.
     Specifies the number of web workers for RMT.
     Acceptable values: Integer greater than or equal to 1.
 
+
+**Container registry settings**
+
+The `registry` setting lets you adjust behavior of the container registry integration.
+
+  * `token_expiration` setting:
+    Specifies the timeout, from issuance, of the authentication/authorization token used by container clients (e.g. podman, docker).
+    Default value is 8 hours.
+    Acceptable values: Integer greater than 1, in seconds.
+
```

Comment:
As the registry auth is a functionality only available from the public cloud engine and not really supported/available for customers with standard RMT package, I'd suggest to move this to the engine's documentation.

Reply:
@digitaltom updated